packages/router/src/entry.route.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/router
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Context} from '@loopback/core';
+
+/**
+ * An entry in the routing table
+ */
+export interface RouteEntry<P, A, R> {
+  /**
+   * Path
+   */
+  readonly path: P;
```

Comment:
Are you sure the `path` is enough to find the route? In a typical HTTP/REST application, we need both the `path` and the `verb` to uniquely identify a route.

I am also not sure if it's needed to parameterize the path type - what use cases do you have in mind where a path would NOT be a `string`?

Maybe we should remove `path` altogether from this first version? We can always add it back later in a backwards-compatible way:

```
// v1
export interface RouteEntry<A, R> { /*..*/ }

// v1.1
export interface RouteEntry<A, R, P=string> {
   readonly path: P;
}
```

At least I think this should be backwards compatible.